test_runner/fixtures/neon_fixtures.py

```diff
@@ -1122,6 +1122,11 @@ def attachment_service_ready():
         # reconcile.
         wait_until(30, 1, attachment_service_ready)
 
+        if register_pageservers:
+            # Special case for forward compat tests, this can be removed later.
```

Comment:
Yes please, super confusing to see a `register_pageservers` parameter added to the test APIs  in this PR.

Reply:
So that we don't forget: https://github.com/neondatabase/neon/pull/7097